events/message.js

```diff
@@ -80,19 +80,26 @@ module.exports = async (config, client, influx, message) => {
         }
     }
     const arg = message.cleanContent.split(' ')
-    if (/^(o|O)+(o|O)(f|F)$/.test(message.cleanContent.split(' '))) {
+    // If message is only some form of "oof" then send the oof file
+    if (/^(o+of)$/ig.test(message.cleanContent)) {
         message.channel.send({
-            files: ['https://cdn.discordapp.com/attachments/437302483044401152/446047008147374091/Roblox_Death_Sound_Effect.mp3']
+            files: ['./Roblox_Death_Sound_Effect.mp3']
```

Comment:
this now uploads it every time is that what you want?

Reply:
Is that a bad thing? I'd worry about the CDN removing reference to the file otherwise.

Reply:
well not bad but it is slow, ive yet to see the CDN remove anything

Reply:
I'll change it back then, and use the CDN for bidoof too